layouts/shortcodes/notices/prototypeShip.html

```diff
@@ -0,0 +1 @@
+{{ partial "notice.html" (merge (dict "page" .Page) (dict "noticeTitle" "Prototype" "noticeDescription" "This Super Capital Ship is classified as a Prototype, a massive, enormously expensive warship with unparalleled firepower and health." "noticeColor" "linear-gradient(270deg, #FC0000, #5300c4)" ) ) }}
```

Comment:
Gradients don't work in the same way as they did in Mediawiki. You have to use a Tailwind CSS class. I recommend setting this to a flat color instead. We *can* do gradients, but I think making the required changes to the notice partial is out of scope for this PR.